js/gateio.js

```diff
@@ -1866,9 +1866,9 @@ module.exports = class gateio extends Exchange {
             type = market['type'];
         } else {
             if (type === 'swap' || type === 'future') {
-                const settle = this.safeStringLower (params, 'settle');
+                const settle = this.safeString (params, 'settle');
```

Comment:
It should be safeStringLower, because that's what's required for the url to work, but our unified currency codes are all uppercase, so needing to pass a lowercase currency here is less unified.